src/cancel.rs

```diff
@@ -180,13 +191,13 @@ impl ::protobuf::Message for Cancel {
         &mut self.unknown_fields
     }
 
-    fn as_any(&self) -> &::std::any::Any {
-        self as &::std::any::Any
+    fn as_any(&self) -> &dyn (::std::any::Any) {
+        self as &dyn (::std::any::Any)
     }
-    fn as_any_mut(&mut self) -> &mut ::std::any::Any {
-        self as &mut ::std::any::Any
+    fn as_any_mut(&mut self) -> &mut dyn (::std::any::Any) {
+        self as &mut dyn (::std::any::Any)
     }
-    fn into_any(self: Box<Self>) -> ::std::boxed::Box<::std::any::Any> {
+    fn into_any(self: Box<Self>) -> ::std::boxed::Box<dyn (::std::any::Any)> {
```

Comment:
Did you run `cargo fmt` on the files? With `edition=2018` the `::prefix` on paths should have been removed.

Reply:
I haven't run anything, just the generation steps from the lib. I could take a look on that later on and try to fix the CI as well